### ISSUE TITLE
rss free broken

### ISSUE THREAD

**Issue by elbenfreund** [pythonjobs/jobs#213]
I am not sure to which extend this is due to github but importing your [feed](http://pythonjobs.github.io/atom.xml) into tiny tiny rss throws the following error ``error 68 at line 623 (column 69), xmlParseEntityRef: no name``.

Please let me know if I can provide additional backup information and thanks for your work on this.

**Comment by salimfadhley** [comment]
yep - it looks like it's not quite right: http://www.feedvalidator.org/check.cgi?url=http%3A%2F%2Fpythonjobs.github.io%2Fatom.xml

@elbenfreund what are you using the RSS feed for?

**Comment by salimfadhley** [comment]
On further examination I can see that the fault is caused by & characters which are not getting escaped properly during our compilation phase. I've fixed the offending job but we need to either fix the validation or improve our compilation so this doesn't happen again. @lordmauve @stestagg, any suggestions?
